src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -174,38 +176,52 @@ const AugmentedRealityView = ({ sceneNavigator }) => {
 
   // TODO: these data can be updated according to the data coming from the server when the
   //       real 3D models arrive
-  const position = [0, -1, -5];
+  const position = [0, 0, 0];
   const rotation = [0, 0, 0];
-  const scale = [0.02, 0.02, 0.02];
+  const scale = [0.002, 0.002, 0.002];
+
+  ViroARTrackingTargets.createTargets({
+    targetImage: {
+      physicalWidth: 0.2, // real world width in meters
+      source: { uri: object.target },
```

Comment:
what happens if `object.target` would be undefined?

Reply:
If undefined, nothing happens. But the object cannot be displayed.

Reply:
updated the code to show the object directly in case there is no target image

https://github.com/ikuseiGmbH/smart-village-app-app/pull/592/commits/cb2f8b030d0be12176bb12acbbc265177dbd7e43